optaplanner-examples/src/test/resources/org/optaplanner/examples/tsp/data/lu980.xml

```diff
@@ -0,0 +1,8830 @@
+        <TspAirLocation id="354">
+            <id>352</id>
+            <latitude>49866.9444</latitude>
+            <longitude>5773.3333</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="355">
+            <id>353</id>
+            <latitude>49954.7222</latitude>
+            <longitude>6106.9444</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="356">
+            <id>354</id>
+            <latitude>50053.3333</latitude>
+            <longitude>6061.9444</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="357">
+            <id>355</id>
+            <latitude>49710.0</latitude>
+            <longitude>6061.6667</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="358">
+            <id>356</id>
+            <latitude>50051.6667</latitude>
+            <longitude>6074.7222</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="359">
+            <id>357</id>
+            <latitude>49702.7778</latitude>
+            <longitude>6426.3889</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="360">
+            <id>358</id>
+            <latitude>49627.5</latitude>
+            <longitude>6023.3333</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="361">
+            <id>359</id>
+            <latitude>49800.0</latitude>
+            <longitude>5866.6667</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="362">
+            <id>360</id>
+            <latitude>49757.7778</latitude>
+            <longitude>6353.3333</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="363">
+            <id>361</id>
+            <latitude>49716.6667</latitude>
+            <longitude>6183.3333</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="364">
+            <id>362</id>
+            <latitude>49789.4444</latitude>
+            <longitude>5839.7222</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="365">
+            <id>363</id>
+            <latitude>49789.4444</latitude>
+            <longitude>5839.7222</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="366">
+            <id>364</id>
+            <latitude>49701.6667</latitude>
+            <longitude>5916.3889</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="367">
+            <id>365</id>
+            <latitude>49810.5556</latitude>
+            <longitude>5918.6111</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="368">
+            <id>366</id>
+            <latitude>49733.3333</latitude>
+            <longitude>6133.3333</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="369">
+            <id>367</id>
+            <latitude>49668.3333</latitude>
+            <longitude>6330.2778</longitude>
+        </TspAirLocation>
+        <TspAirLocation id="370">
+            <id>368</id>
+            <latitude>49700.0</latitude>
+            <longitude>5983.3333</longitu
```

Comment:
There seems to be an identical file in `optaplanner-examples/data/tsp/unsolved`. Can we reuse it?